.github/workflows/serverless-pre-release-qa.yml

```diff
@@ -0,0 +1,139 @@
+# Workflow to create a new pre-release with qa suffix
+
+name: Pre-release QA
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+    inputs:
+      tag-prefix:
+        description: Release tag name.
```

Comment:
What exactly needs to be provided here? The version and the prerelease tag name (e.g. `1.5.1-alpha`)?

Reply:
Sorry the poor description, I've updated it!